hasjob/models/jobpost.py

```diff
@@ -429,6 +430,11 @@ def viewcounts_key(jobpost_id):
             return ['hasjob/viewcounts/%d' % post_id for post_id in jobpost_id]
         return 'hasjob/viewcounts/%d' % jobpost_id
 
+    @staticmethod
+    def maxcounts_key(postids):
+        return ['hasjob/maxcounts/{digest}'.format(digest=
+            hashlib.sha1(','.join(str(postid) for postid in postids)).hexdigest())]
```

Comment:
I thought we removed the need for this method.

Reply:
We did, but we later discussed caching `get_max_counts` since it could result in 700 function calls to `get_post_viewcounts`.